src/main/resources/application.properties

```diff
@@ -54,7 +54,7 @@ endpoint.company_name_mongo_alpha.timer=timer:company_name_mongo_alpha?repeatCou
 endpoint.company_status_mongo_primary.timer=timer:company_status_mongo_primary?repeatCount=1&delay=${COMPANY_STATUS_MONGO_PRIMARY_DELAY}
 endpoint.company_status_mongo_alpha.timer=timer:company_status_mongo_alpha?repeatCount=1&delay=${COMPANY_STATUS_MONGO_ALPHA_DELAY}
 endpoint.company_status_mongo_oracle.timer=timer:company_status_mongo_oracle?repeatCount=1&delay=${COMPANY_STATUS_MONGO_ORACLE_DELAY}
-endpoint.insolvency.company_number.timer=timer:insolvency_company_number?repeatCount=1&delay=${INSOLVENCY_COMPANY_NUMBER_MONGO_ORACLE_DELAY}
+endpoint.insolvency.company_number.mongo_oracle.timer=timer:insolvency_company_number?repeatCount=1&delay=${INSOLVENCY_COMPANY_NUMBER_MONGO_ORACLE_DELAY}
```

Comment:
Incorrect period separator: company_number.mongo_oracle should read company_number_mongo_oracle